app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/homescreen/HomeScreen.kt

```diff
@@ -0,0 +1,562 @@
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.Top,
+            modifier =
+            modifier
+                .fillMaxWidth()
+                .padding(
+                    horizontal = dimensionResource(R.dimen.home_screen_horizontal_padding),
+                ),
+        ) {
+            Text(
+                text = "What do you want to\nwatch today?",
+                style = MaterialTheme.typography.titleLarge,
+                textAlign = TextAlign.Start,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.height(height = dimensionResource(R.dimen.home_screen_headline_text_height)),
+            )
+            Box(
+                modifier =
+                Modifier
+                    .padding(top = dimensionResource(R.dimen.home_screen_headline_image_padding_top))
+                    .size(dimensionResource(R.dimen.home_screen_headline_image_size))
+                    .background(MaterialTheme.colorScheme.onBackground)
+                    .clip(shape = CircleShape)
+                    .border(
+                        border =
+                        BorderStroke(
+                            width = dimensionResource(R.dimen.home_screen_headline_image_border_width),
+                            color = MaterialTheme.colorScheme.onSurface
+                        ),
+                        shape = CircleShape,
+                    ),
+            ) {
+                Image(
+                    painter = painterResource(R.drawable.profile_picture),
+                    contentDescription = stringResource(R.string.profile_picture_content_description),
+                    contentScale = ContentScale.FillBounds,
+                )
+            }
+        }
+}
+
+@SuppressLint("UseOfNonLambdaOffsetOverload")
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun HomeScreenSearchBar(modifier: Modifier = Modifier) {
+    var query by rememberSaveable { mutableStateOf("") }
+    var expanded by rememberSaveable { mutableStateOf(false) }
+    val suggestions =
+        rememberSaveable {
+            mutableListOf(
+                "Spider-man 2",
+                "Batman",
+            )
+        }
+
+    val animatedDp by animateDpAsState(
+        targetValue = if (expanded) 0.dp else dimensionResource(R.dimen.home_screen_search_horizontal_padding),
+        animationSpec =
+        tween(
+            durationMillis = 400,
+            easing = FastOutLinearInEasing,
+        ),
+        label = "",
+    )
+
+    val animatedYaxisOffset by animateDpAsState(
+        targetValue = if (expanded) 0.dp else dimensionResource(R.dimen.home_screen_search_bar_y_offset),
+        animationSpec =
+        tween(
+            durationMillis = 400,
+            easing = FastOutLinearInEasing,
+        ),
+        label = "",
+    )
+
+    Box(
+        modifier = modifier.fillMaxSize(),
```

Comment:
Set fillMaxSize() in the calling composable

Reply:
done